### ISSUE TITLE
Safari Cannot use initialiser syntax in a strict mode enumeration.

### ISSUE THREAD

**Issue by zoe-1** [google/traceur-compiler#1862]
Followed the tutorial learned about from Brad Greens twitter post today. 
I was using Safari and got errors:
Made a git repo describing the issue.
 
You can see all the files I used there.
The README file explains it.
Visit: https://github.com/zoe-1/angular2-issue
I hope this helps.

ERROR MESSAGES
1st message ----
[Error] SyntaxError: Cannot use initialiser syntax in a strict mode enumeration.
    (anonymous function) (es6-shim.js, line 4941)

2nd message ----
[Error] ReferenceError: Can't find variable: System
    global code (index.html, line 14)

**Comment by johnjbarton** [comment]
es6-shim dies in Safari on 
```
[Error] SyntaxError: Cannot use initialiser syntax in a strict mode enumeration.
	(anonymous function) (es6-shim.js, line 4941)
```
```js
for (var name = void 0 in this.variableDeclarations) {
                    names[name] = true;
                }
```

**Comment by johnjbarton** [comment]
So you were correct, the problem is a duplicate of issue #1647.

**Comment by johnjbarton** [comment]
We shipped the fix about three weeks ago, in version 0.0.87.  Looks like the copy of es6-shim in the quickstart is 0.0.82.  I'm unsure how hard it is to install an upgrade into es6-shim

**Comment by zoe-1** [comment]
Thank you for the es6-shim recommendation.
